build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md

```diff
@@ -75,6 +75,14 @@ Below is an example for ads.
 
 [/example][/filter]
 
+### Standalone use outside the AMP environment (optional)
+
+Bento AMP allows you to use AMP components in non-AMP pages without needing to commit to fully valid AMP. You can take these components and place them in implementations with frameworks and CMSs that don't support AMP. Read more in our guide [Use AMP components in non-AMP pages](https://amp.dev/documentation/guides-and-tutorials/start/bento_guide/)
```

Comment:
Should we link "Bento AMP" to the guide?

Reply:
Yessir! The last sentence links to the guide